README.es-ES.md

```
@@ -0,0 +1,131 @@
+
+Consulta la documentación oficial para más ideas.
+
+## ✍️ Documentación Oficial & Otros Usos & Preguntas a IA
+
+> [👉 Haz clic para entrar](https://zhiyu1998.github.io/rconsole-plugin/)
+
+Para consultas, puedes preguntar al agente IA `GLM` que creé:
+
+<img src="https://s2.loli.net/2024/10/12/eDi6FcYITkMUpSb.png" style="zoom: 33%;" >
+
+##  🤺 Grupo de Chat del Plugin R
```

markdownlint-cli2 (0.23.2)

[warning] Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)